frame/session/src/historical/offchain.rs

```diff
@@ -138,14 +139,83 @@ pub fn prove_session_membership<T: Trait, D: AsRef<[u8]>>(
 
 #[cfg(test)]
 mod tests {
-    use super::super::tests::new_test_ext;
+    use super::super::tests;
+    use super::super::{onchain,Module};
```

Comment:
spaces used to indent in this file should be replaced with tabs

Reply:
I might need to tweak the `rustfmt.toml` some more then..

Reply:
🤔